Geometry/CaloGeometry/interface/CaloCellGeometry.h

```diff
@@ -139,7 +142,13 @@ class CaloCellGeometry
   }
 
   virtual void initCorners(CornersVec&) = 0;
+
+  GlobalPoint         m_refPoint ;
```

Comment:
CMS coding convention says member data should only be private. What was the purpose to move these to protected?